var/spack/repos/builtin/packages/py-griddataformats/package.py

```diff
@@ -15,12 +15,39 @@ class PyGriddataformats(PythonPackage):
     homepage = "http://www.mdanalysis.org/GridDataFormats"
     pypi = "GridDataFormats/GridDataFormats-0.5.0.tar.gz"
 
-    version("0.5.0", sha256="f317ed60708de22d1b2a76ce89a00f722d903291b1055ff1018d441870c39d69")
-    version("0.4.1", sha256="b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3")
-    version("0.4.0", sha256="f81d6b75aa7ebd9e8b64e14558c2d2583a0589829382beb4ef69860110261512")
-    version("0.3.3", sha256="938f0efcb3bc2f58ec85048b933942da8a52c134170acc97cb095f09d3698fbd")
+    maintainers("RMeli")
+
+    version("1.0.1", sha256="ad2c9ab7d672a6d8c426de7d083eee4f3e2b0bd59391675d30683c768ab83cc4")
+    version(
+        "0.5.0",
+        sha256="f317ed60708de22d1b2a76ce89a00f722d903291b1055ff1018d441870c39d69",
+        deprecated=True,
+    )
+    version(
+        "0.4.1",
+        sha256="b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3",
+        deprecated=True,
+    )
+    version(
+        "0.4.0",
+        sha256="f81d6b75aa7ebd9e8b64e14558c2d2583a0589829382beb4ef69860110261512",
+        deprecated=True,
+    )
+    version(
+        "0.3.3",
+        sha256="938f0efcb3bc2f58ec85048b933942da8a52c134170acc97cb095f09d3698fbd",
+        deprecated=True,
+    )
 
-    depends_on("python@2.7:")
     depends_on("py-setuptools", type="build")
-    depends_on("py-numpy@1.0.3:", type=("build", "run"))
-    depends_on("py-six", type=("build", "run"))
+
+    depends_on("python@3.8:", when="@1.0.1:", type=("build", "run"))
+
+    depends_on("py-numpy@1.19:", when="@1.0.1", type=("build", "run"))
+    depends_on("py-scipy", when="@1.0.1", type=("build", "run"))
+    depends_on("py-mrcfile", when="@1.0.1", type=("build", "run"))
+
+    # Deprecated
+    depends_on("python@2.7:", when="@:0.5.0", type=("build", "run"))
+    depends_on("py-numpy@1.0.3:", type=("build", "run"), when="@:0.5.0")
+    depends_on("py-six", type=("build", "run"), when="@:0.5.0")
```

Comment:
```suggestion
    depends_on("python@3.8:", when="@1:", type=("build", "run"))

    depends_on("py-numpy@1.19:", when="@1:", type=("build", "run"))
    depends_on("py-scipy", when="@1:", type=("build", "run"))
    depends_on("py-mrcfile", when="@1:", type=("build", "run"))

    # Deprecated
    depends_on("python@2.7:", when="@0", type=("build", "run"))
    depends_on("py-numpy@1.0.3:", when="@0", type=("build", "run"))
    depends_on("py-six", when="@0", type=("build", "run"))
```
Version 1.0.0 was when these changed, presumably these will also be needed in 1.0.2+

Reply:
Yes, the lack of `:` was an oversight, thanks for catching it.